### ISSUE TITLE
Move reaction-collections to NPM

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#4107]
Add per @vwadher 

Move this package to npm so that packages that depend on it can just depend pn the npm package rather than needing Meteor dependency injection.

**Comment by zenweasel** [comment]
@jshimko  Was this already done or?

**Comment by jshimko** [comment]
Yes, this has been done in #4149.  Just waiting on final review/merge.

**Comment by zenweasel** [comment]
Closed via #4149
